SimpleServlet/src/main/java/com/skankhunt220/repository/UserRepository.java

```diff
@@ -0,0 +1,56 @@
+package main.java.com.skankhunt220.repository;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import org.bson.Document;
+import org.bson.types.ObjectId;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientException;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import main.java.com.skankhunt220.entities.User;
+
+public class UserRepository {
+	private MongoClient mc;
+	private MongoDatabase database;
+	private MongoCollection<Document> collection;
+
+	public UserRepository() {
+		mc = new MongoClient("localhost", 27017);
+		database = mc.getDatabase("UserDB");
+		collection = database.getCollection("users");
+	}
+
+	public void Create(User user) {
```

Comment:
Folow method convetion in java the methods should be verbs, in mixed case with the first letter lowercase and with the first letter of each internal word capitalised.